<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Genre:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Type:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="r_homme">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>119</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Homme</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="r_femme">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>119</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Femme</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter Client</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inp_id">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="ch_import">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>100</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Import</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ch_export">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>100</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_select">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>220</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher Client</string>
    </property>
   </widget>
   <widget class="QComboBox" name="com_attributs">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>86</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Tous</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Homme</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Femme</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Import</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Export</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Import/Export</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="table_result">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>301</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Genre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_reset">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>290</y>
      <width>97</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
